www/klei00/hw3/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+
+<body>
+    <div class="task1">The font of this web page is not serif</div>
```

Comment:
u těch názvů tříd a id doporučuju psát s pomlčkou, tj. task-1, task-12, pro vývojáře je to přehlednější